Cargo.toml

```
@@ -2,5 +2,5 @@
 
 members = [
   "nats-legacy",
```

As you brought up, rename back 🙂 

```suggestion
  "nats",
```